Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,142 @@
+                var weight = PromptFloat("What is your weight in kilograms");
+                var heightCm = PromptFloat("What is your height in centimeters");
+                var bmi = CalculateBmi(weight, heightCm / 100);
+
+                Console.WriteLine();
+                Console.WriteLine("Name: " + name + " " + surname);
+                Console.WriteLine("Age: " + age);
+                Console.WriteLine("Weight: " + weight + "kg");
+                Console.WriteLine("Height: " + heightCm / 100 + "m");
+                Console.WriteLine("BMI: " + bmi);
+
+
+                Console.WriteLine(Environment.NewLine + "Continue with a new set of inputs? " +
+                "Press Y to continue or press any other button to exit the program" + Environment.NewLine);
+
+                string answer = Console.ReadLine().ToLowerInvariant();
+                if (answer == "y")
+                {
+                    Console.WriteLine(Environment.NewLine + "Please enter your data" + Environment.NewLine);
+                }
+                else
+                {
+                    Console.WriteLine(Environment.NewLine + "Thank you. Good Bye");
+                    break;
+                }
+            }
+            while (true);
+
+        }
+
+        public static string PromptString(string message)
+        {
+            Console.WriteLine(message);
+            var GrabString = Console.ReadLine();
+            var StringCheck = string.IsNullOrWhiteSpace(GrabString);
+            if (StringCheck == true)
+            {
+                Console.WriteLine("Name cannot be empty");
+                return "-";
+            }
+            else
+            {
+                return GrabString;
+            }
+        }
+
+        public static int PromptInt(string message)
+        {
+            Console.WriteLine(message);
+            var GrabInt = Console.ReadLine();
+            //Checks if input of age is empty or not
+            var IntNumberCheck = string.IsNullOrWhiteSpace(GrabInt);
+            //Checks if input is a valid number or not
+            if (IntNumberCheck != true)
+            {
+                bool CheckValidNumber = int.TryParse(GrabInt, out int age);
+                if (CheckValidNumber == true)
+                {
+                    return age;
+                }
+                else
+                {
+                    Console.WriteLine(GrabInt + " is not a valid number");
+                    return -1;
+                }
+            }
+            else
+            {
+                return 0;
+            }
+        }
+
+        public static float PromptFloat(string message)
+        {
+            Console.WriteLine(message);
+            var GrabFloat = Console.ReadLine();
+            //Checks if input of age is empty or not
+            var FloatNumberCheck = string.IsNullOrWhiteSpace(GrabFloat);
+            //Checks if input is a valid number or not
+            if (FloatNumberCheck != true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4913227)